Northcoders/projects/fe-nc-news/Api.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'
+
+const marketPlaceAPI = axios.create({
+    baseURL: 'https://nc-news-4rxh.onrender.com/api'
+})
+
+export function fetchAllArticles() {
+    return marketPlaceAPI.get("/articles")
+    .then((response) => {
+        return response.data
+    })
+}
+
+export function fetchArticleId(article_id) {
```

Comment:
Just for consistency when someone looks at your code - I would keep your res and response the same throughout functions